### ISSUE TITLE
Non-numeric error when using purrr at tract level

### ISSUE THREAD

**Issue by oooandrade** [walkerke/tidycensus#156]
When using `purrr` package's `map_df` feature, described in the DataCamp tutorials, it doesn't appear to work at the tract level unless a county is supplied to the argument. However, you don't need a county specified when running the function outside of the `map_df` wrapper. See code below.

This works:

```
mi_tracts <- purrr::map_df(c(2012, 2014), function(x) {
  get_acs(geography = "tract", 
          variables = c(totalpop = "B01003_001"), 
          state = "MI", 
          county = "Iosco",
          survey = "acs5", 
          year = x) %>%
    mutate(year = x)
})
```

This fails with the error below:
```
mi_cities_v2 <- map_df(c(2012, 2014), function(x) {
  get_acs(geography = "tract", 
          variables = c(totalpop = "B01003_001"), 
          state = "MI", 
          survey = "acs5", 
          year = as.numeric(x)) %>%
    mutate(year = x)
})

> Error in year - 4 : non-numeric argument to binary operator 
```

However, this works:
```
get_acs(geography = "tract", 
        variables = c(totalpop = "B01003_001"), 
        state = "MI", 
        survey = "acs5", 
        year = 2012) %>%
  mutate(year = 2012)
```

**Comment by mfherman** [comment]
Hi @oooandrade. This is known issue with how `get_acs()` internally handles getting tracts from multiple counties for multiple years. Take a look at #136  and #145 for more detailed discussion as well as some workarounds until this is fixed in `get_acs()`.

**Comment by walkerke** [comment]
This is now fixed in tidycensus master (0.9.2).
